apps/controller_info.py

```diff
@@ -56,6 +60,38 @@ def command_succeeded(response):
     return False
 
 
+# -----------------------------------------------------------------------------
+async def get_common_info(host):
+    if host.supports_command(HCI_READ_LOCAL_SUPPORTED_FEATURES_COMMAND):
+        response = await host.send_command(
+            HCI_Read_Local_Supported_Features_Command()
+        )
+        if response.return_parameters.status == HCI_SUCCESS:
+            print()
+            print(color('LMP Features:', 'yellow'))
+            # TODO: support printing discrete enum values
+            print('  ', response.return_parameters.lmp_features.hex())
```

Comment:
I think it would be helpful to parse features into `set` or `IntFlag` or something else because sometimes we need to check controller capabilities for both dev and product evaluation.

Reply:
I agree. I'll try to add that in a separate PR.